fe/fe-core/src/main/java/org/apache/doris/ha/MasterInfo.java

```diff
@@ -27,17 +29,20 @@
 public class MasterInfo implements Writable {
 
     private String ip;
+    private String hostName;
```

Comment:
The hostName maybe null, we need to handle it.
And you can change the serde to GSON for this class.

Reply:
And it is every error prone to be null for a field.
How about keep it as `empty string` is not set, and use `Strings.isEmpty()` to check it when use it?
Same suggestion for host in Frontend.java